### DISCUSSION TITLE
Docker coldfusion 2023

### DISCUSSION THREAD

**Discussion by michaeldiederich** [wheels-dev/wheels#1280] [General]
The docker setup has a container with Adobe 2016 and Adobe 2018.
There is a docker with adobe 2023 at https://hub.docker.com/u/adobecoldfusion

Is there any reason, that it is not setup and in use for the tests? Is nobody using it?

**Comment by michaeldiederich** [comment]
Notice: Found this Pullrequest for Adobe 2021

https://github.com/cfwheels/cfwheels/pull/1230

**Comment by bpamiri** [comment]
Still having issues with Adobe package manager to install the required packages. That is what is holding up testing with the latest Adobe engines.Sent from my iPad-Peter Amiri m 949 338 3862On Sep 25, 2023, at 6:21 AM, Michael Diederich ***@***.***> wrote:﻿
Notice: Found this Pullrequest for Adobe 2021
#1230

**Comment by bpamiri** [comment]
I finally got some time to work on this a little more. Happy to say that both the local Docker based test suite and the Github Actions test runners are all updated and now test Lucee 5 and 6 against MySQL, SQL Server, PostgreSQL, and H2 databases and on the Adobe side Adobe ColdFusion 2016, 2018, 2021, and 2023 are tested against MySQL, SQL Server, and PostgreSQL. 

To test the latest build you can use the command:
`box install cfwheels-base-template@2.5.0-dev`
